app/assets/stylesheets/components/_option-select.scss

```diff
@@ -169,16 +97,87 @@
   fill: currentColor;
 }
 
-.js-enabled .app-c-option-select__icon--up {
-  display: block;
+.app-c-option-select__container {
+  position: relative;
+  height: 200px;
+  background-color: $page-colour;
+  overflow-y: auto;
+  overflow-x: hidden;
+}
+
+.app-c-option-select__container-inner {
+  padding: $gutter-one-quarter;
 }
 
-.js-closed {
+.app-c-option-select__filter {
+  display: none;
+
+  .govuk-form-group {
+    margin-bottom: $gutter-one-third;
```

Comment:
Could we use govuk static spacing https://design-system.service.gov.uk/styles/spacing/#using-the-static-spacing-function